src/exports.jl

```diff
@@ -71,6 +71,7 @@ export AbstractITensorNetwork,
   mps,
   ortho_center,
   set_ortho_center,
+  BeliefPropagationCache,
```

Comment:
Let's not export it for now, if anything I would prefer to not export anything new and reconsider all of the exports we have right now.